src/OrangePaymentSlipData.php

```diff
@@ -176,7 +207,8 @@ public function getBankingCustomerId()
         if (!$this->getWithBankingCustomerId()) {
             throw new DisabledDataException('banking customer ID');
         }
-        return $this->bankingCustomerId;
+        
+        return str_pad($this->bankingCustomerId, $this->bankingCustomerIdLength, '0', STR_PAD_LEFT);
```

Comment:
The goal to set the `bankingCustomerIdLength` is to be sure that the returned value will have this length including zeroes left padding if necessary. If you provide a `bankingCustomerId` equals to `1234` and a `bankingCustomerIdLength` equals to `6`, you expect `001234` otherwise the `bankingCustomerIdLength` has to be `4`.